R/multitables.R

```diff
@@ -85,25 +85,32 @@ setMethod("is.public<-", "Multitable", function (x, value) {
 #' name(m) # [1] "gender + age4 + marstat"
 #' }
 #' @export
-newMultitable <- function (formula, data, name, ...) {
-    ## Validate inputs
-    if (missing(formula)) {
-        halt("Must provide a formula")
-    }
-    if (missing(data) || !is.dataset(data)) {
-        halt(dQuote("data"), " must be a Dataset")
-    }
+newMultitable <- function (formula, data, name, multitable=NULL, ...) {
+
+    if (missing(multitable)){
+        ## Validate inputs
+        if (missing(formula)) {
+            halt("Must provide a formula")
+        }
+        if (missing(data) || !is.dataset(data)) {
+            halt(dQuote("data"), " must be a Dataset")
+        }
+
+        template <- formulaToQuery(formula, data)
+        if (missing(name)) {
+            name <- formulaRHS(formula)
+        }
 
-    template <- formulaToQuery(formula, data)
-    if (missing(name)) {
-        name <- formulaRHS(formula)
+        payload <- wrapEntity(
+            name=name,
+            template=lapply(template$dimensions,
+                function (x) list(query=x, variable=findVariableReferences(x)))
+        )
+    } else {
+        payload = wrapEntity(name=name,
```

Comment:
Use `<-` for assignment in R. 

And is "name" a required attribute of this payload? I'd expect it would/should use the name of the multitable I'm copying from.